### PR TITLE
 Support for conditional storage of BGP IPv4 and IPv6 Unicast Learned…

### PR DESCRIPTION
**Pull request by apratimmukherjee**
Two new optional flags added in BGP peer to control whether Unicast IPv4 /IPv6 routes should be stored or not. ( Default is : dont store routes )

### CODE REVIEW

File: device/bgp/bgplearnedinfofilter.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+components:
+  schemas:
+    Bgp.LearnedInformationFilter:
+      x-status: under-review
+      description: >-
+        Configuration for controlling storage of BGP learned information recieved from the DUT.
```
Comment (ashutshkumr) [inline device/bgp/bgplearnedinfofilter.yaml:6]: shouldn't it be `peer` instead of `DUT` ?

**Review by ashutshkumr — APPROVED** [review]

**Review by PrasenjitAdhikary — APPROVED** [review]
Looks good to me.

File: device/bgp/bgplearnedinfofilter.yaml (hunk 1 above)
Comment (apratimmukherjee) [reply device/bgp/bgplearnedinfofilter.yaml]: Updated to peer from DUT .

**Review by SuryyaKrJana — APPROVED** [review]
Look good

**Pull request merged**